gulpfile.js

```diff
@@ -103,8 +103,11 @@ gulp.task('copy:main.css', function () {
 
     return gulp.src(dirs.src + '/css/main.css')
                .pipe(plugins.header(banner))
-               .pipe(gulp.dest(dirs.dist + '/css'));
-
+                .pipe(plugins.autoprefixer({
+                    browsers: ['last 2 versions', 'ie >= 8'],
```

Comment:
Related to this discussion: [The Opera Mini servers where upgraded recently!](https://dev.opera.com/blog/opera-mini-server-upgrade/)